sup3r/postprocessing/file_handling.py

```diff
@@ -350,7 +355,8 @@ def write_output(cls, data, features, low_res_lat_lon,
                 flat_data = data[..., i].reshape((-1, len(times)))
                 flat_data = np.transpose(flat_data, (1, 0))
                 Outputs.add_dataset(out_file, f, flat_data,
-                                    dtype=attrs['dtype'], attrs=attrs)
+                                    dtype=attrs['dtype'], attrs=attrs,
+                                    chunks=chunks)
```

Comment:
okay so i dont think chunks should be an input to this method, but instead be part of the H5_ATTRS lookup so you can set the chunk shape per dataset/dtype. So something like windspeed will always be uint16 with chunk shape (2000, 500) which is 2MB. So right here you would just have `chunks=attrs.get('chunks', DEFAULT_CHUNKS)`

Also this might not be fun but you should rebase on my branch after i merge since i made some important edits to the file handlers.

Reply:
Looks like you did exactly this, nevermind!